en/docs/design/api-collaborations/comment-on-api.md

```diff
@@ -0,0 +1,23 @@
+#### Comment on an API
+
```

Comment:
Let's move and rename all the images and img folders as follows:

en/docs/assets/img/assets/img/consume/community-features/apis-select-devportal.png

en/docs/assets/img/assets/img/consume/community-features/devportal-api-overview-select-rating.png

en/docs/assets/img/assets/img/consume/community-features/devportal-api-already-rated.png

en/docs/assets/img/assets/img/consume/community-features/apis-select-devportal.png

en/docs/assets/img/consume/community-features/devportal-add-new-comment.png

en/docs/assets/img/assets/img/consume/community-features/developer-portal-reply-to-comment.png




en/docs/assets/img/assets/img/design/community-features/select-publisher-api.png

en/docs/assets/img/assets/img/design/community-features/publisher-reply-to-comment.png

en/docs/assets/img/assets/img/design/community-features/publisher-reply-to-comment.png




en/docs/assets/img/assets/img/design/community-features/devportal-default-community-links.png

en/docs/assets/img/assets/img/design/community-features/publisher-community-link-basic-info.png

en/docs/assets/img/assets/img/design/community-features/publisher-slack-urls.png

en/docs/assets/img/assets/img/design/community-features/devportal-community-links-with-slack-github.png